.cirrus.yml

```diff
@@ -179,7 +179,7 @@ task:
   build_script:
     - '%x64_NATIVE_TOOLS%'
     - cd %CIRRUS_WORKING_DIR%
-    - ccache --zero-stats
+    - ccache --zero-stats --max-size=%CCACHE_SIZE%
```

Comment:
is `CCACHE_SIZE` set at all?

Reply:
https://github.com/bitcoin/bitcoin/blob/0dd34773334c7f4db7b05df30ee61b011795b46d/.cirrus.yml#L6

Reply:
On [master](https://api.cirrus-ci.com/v1/task/5018066990923776/logs/build.log):
```
Summary:
  Hits:             239 /  239 (100.0 %)
    Direct:         239 /  239 (100.0 %)
    Preprocessed:     0 /    0
  Misses:             0
    Direct:           0
    Preprocessed:     0
  Uncacheable:       10
Primary storage:
  Hits:             478 /  478 (100.0 %)
  Misses:             0
  Cache size (GB): 1.35 / 5.00 (27.06 %)

Use the -v/--verbose option for more details.
```

With [this PR](https://api.cirrus-ci.com/v1/task/5900879822848000/logs/build.log):
```
Summary:
  Hits:             239 /  239 (100.0 %)
    Direct:         239 /  239 (100.0 %)
    Preprocessed:     0 /    0
  Misses:             0
    Direct:           0
    Preprocessed:     0
  Uncacheable:       10
Primary storage:
  Hits:             478 /  478 (100.0 %)
  Misses:             0
  Cache size (GB): 0.07 / 0.20 (32.93 %)

Use the -v/--verbose option for more details.
```

Compare the "Cache size" string.